### PR TITLE
Fix invalid CONSENT cookie value

### PR DESCRIPTION
**Pull request by TobiGr**
- [x] I carefully read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md) and agree to them.
- [x] I have tested the API against [NewPipe](https://github.com/TeamNewPipe/NewPipe).
- [x] I agree to create a pull request for [NewPipe](https://github.com/TeamNewPipe/NewPipe) as soon as possible to make it compatible with the changed API.

### CODE REVIEW

File: extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/YoutubeParsingHelper.java [hunk 1]
```diff
@@ -734,7 +734,8 @@ public static void addCookieHeader(final Map<String, List<String>> headers) {
     }
 
     public static String generateConsentCookie() {
-        return CONSENT_COOKIE + 100 + numberGenerator.nextInt(900);
+        final int statusCode = 100 + numberGenerator.nextInt(900);
+        return CONSENT_COOKIE + statusCode;
```
Comment (mhmdanas) [inline extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/YoutubeParsingHelper.java:738]: How is the new code different from the old?
<sub>Please don't kill me if it's obvious</sub>

Comment (TobiGr) [reply extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/YoutubeParsingHelper.java:738]: `CONSENT_COOKIE` is a String. That's why the result looks like `CONSENT=100580`.

Comment (mhmdanas) [reply extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/YoutubeParsingHelper.java:738]: So if I understand correctly, an equivalent change would be keeping the method with one statement, but wrapping `100 + numberGenerator.nextInt(900)` in parenthesis, right?

**Review by TiA4f8R — APPROVED** [review]
Thank you for fixing this bug!

**Pull request merged**